sdks/java/core/src/test/java/org/apache/beam/sdk/options/ProxyInvocationHandlerTest.java

```diff
@@ -236,48 +236,17 @@ public void testEqualsAndHashCode() throws Exception {
   }
 
   @Test
-  public void testToString() throws Exception {
-    ProxyInvocationHandler handler = new ProxyInvocationHandler(Maps.<String, Object>newHashMap());
-    Simple proxy = handler.as(Simple.class);
-    proxy.setString("stringValue");
-    DefaultAnnotations proxy2 = proxy.as(DefaultAnnotations.class);
-    proxy2.setLong(57L);
-    assertEquals("Current Settings:\n"
-        + "  long: 57\n"
-        + "  string: stringValue\n",
-        proxy.toString());
-  }
-
-  @Test
-  public void testToStringAfterDeserializationContainsJsonEntries() throws Exception {
-    ProxyInvocationHandler handler = new ProxyInvocationHandler(Maps.<String, Object>newHashMap());
-    Simple proxy = handler.as(Simple.class);
-    Long optionsId = proxy.getOptionsId();
-    proxy.setString("stringValue");
-    DefaultAnnotations proxy2 = proxy.as(DefaultAnnotations.class);
-    proxy2.setLong(57L);
-    assertEquals(String.format("Current Settings:\n"
-        + "  long: 57\n"
-        + "  optionsId: %d\n"
-        + "  string: \"stringValue\"\n", optionsId),
-        serializeDeserialize(PipelineOptions.class, proxy2).toString());
-  }
-
-  @Test
-  public void testToStringAfterDeserializationContainsOverriddenEntries() throws Exception {
-    ProxyInvocationHandler handler = new ProxyInvocationHandler(Maps.<String, Object>newHashMap());
-    Simple proxy = handler.as(Simple.class);
-    Long optionsId = proxy.getOptionsId();
-    proxy.setString("stringValue");
-    DefaultAnnotations proxy2 = proxy.as(DefaultAnnotations.class);
-    proxy2.setLong(57L);
-    Simple deserializedOptions = serializeDeserialize(Simple.class, proxy2);
-    deserializedOptions.setString("overriddenValue");
-    assertEquals(String.format("Current Settings:\n"
-        + "  long: 57\n"
-        + "  optionsId: %d\n"
-        + "  string: overriddenValue\n", optionsId),
-        deserializedOptions.toString());
+  public void testMultipleDefaultsThrow() throws Exception {
```

Comment:
All three tests becomes illegal (cast between Simple.class and DefaultAnnotations.class).

So, only one is needed.